lib/helpers.js

```diff
@@ -92,7 +92,8 @@ _beforeEach.givenModel = function(modelName, attrs, optionalHandler) {
 
   beforeEach(function(done) {
     if(modelName === '__USERMODEL__') {
-      modelName = this.userModel ? this.userModel : 'user';
+      modelName = this.userModel ? this.userModel : 'User';
```

Comment:
@clarkorz can you please clarify further. In this case we are interested in simply getting the name(ie string, to use as a key to lookup subsequently) of the `User` model, and don't really need the actual *Model* or the *Type*.